src/Consolonia.Gallery/View/ControlsListView.axaml

```diff
@@ -63,15 +63,18 @@
             </MenuItem>
         </Menu>
 
-        <DataGrid x:Name="GalleryGrid"
-                  DockPanel.Dock="Left"
-                  SelectedIndex="0"
-                  SelectionMode="Single">
-            <DataGrid.Columns>
-                <DataGridTextColumn Header="Name"
-                                    Binding="{Binding Name}" />
-            </DataGrid.Columns>
-        </DataGrid>
+        <ListBox x:Name="GalleryGrid"
+                 DockPanel.Dock="Left"
+                 SelectedIndex="0"
+                 Focusable="true"
+                 Width="24"
+                 SelectionMode="Single">
+            <ListBox.ItemTemplate>
+                <DataTemplate>
```

Comment:
This is probably default template of the ListBox?

Reply:
Um. I need to bind to Name. How do I do that without an ItemTemplate?